webview/src/stories/Setup.stories.tsx

```diff
@@ -42,7 +56,13 @@ export default {
 } as Meta
 
 const Template: StoryFn = ({ data }) => {
-  const app = <App />
+  const app = (
+    <Provider store={configureStore({ reducer: setupReducers })}>
```

Comment:
You can pass a preloaded state here instead of using dispatch and feeding the store like you currently do. The good part about this is that the story is ready on the get-go, it's not going to change while the store is being fed. Also, you won't need to export stuff from the webview simply for the story.

```
<Provider
      store={configureStore({
        preloadedState: { ...data },
        reducer: experimentsReducers
      })}
    >
```

Reply:
On further inspection, we'd need to adjust our `DEFAULT_DATA` to be configured like our redux state object for us to pass it as our `preloadedState`. While we could do that, it would make the `DEFAULT_DATA` harder to adjust with its nested state objects. I'll keep the `MockedState` component for now to make story creation simpler.